NDB.Covid19/NDB.Covid19.iOS/Views/MessagePage/MessagePageViewController.cs

```diff
@@ -64,11 +65,22 @@ public override void ViewWillDisappear(bool animated)
             UIApplication.SharedApplication.ApplicationIconBadgeNumber = 0;
         }
 
-        private void SetStyling()
+        private async void SetStyling()
         {
             StyleUtil.InitLabelWithSpacing(Label, StyleUtil.FontType.FontBold, MESSAGES_HEADER, 1.14, 22, 34);
-            StyleUtil.InitLabelWithSpacing(LabelLastUpdate, StyleUtil.FontType.FontRegular, LastUpdateString, 1.14, 12, 14, UITextAlignment.Left);
-            StyleUtil.InitLabelWithSpacing(NoItemsLabel1, StyleUtil.FontType.FontBold, MESSAGES_NO_ITEMS_TITLE, 1.25, 32, 34);
+            StyleUtil.InitLabelWithSpacing(LabelLastUpdate, StyleUtil.FontType.FontRegular, LastUpdateString, 1.14, 15, 17, UITextAlignment.Left);
+            int unreadMessages = (await MessageUtils.GetAllUnreadMessages()).Count;
+            int messages = (await MessageUtils.GetMessages()).Count;
+            string headerText = MESSAGES_NO_ITEMS_TITLE;
+            if (unreadMessages > 0)
+            {
+                headerText = MESSAGES_NEW_MESSAGES_HEADER;
+            }
+            else if (messages > 0)
```

Comment:
What if messages is 0?

Reply:
Line 74

Reply:
aha, then it is default. yeah, I got it :D